development/includes/blog-column.pug

```diff
@@ -0,0 +1,61 @@
+//- <!-- Blog preview link boxes -->
+div.blog-previews
```

Comment:
This should be structure more semantically (using meaningful tags, instead of tags that work). Please remove this div and replace with a nav element. The structure for this menu should looks something like this: 

```
ul.blog-previews
  li.blog-preview-box
    a(href="/blog/chapter1htmlcss.html")
      img.blog-preview-image(src="/resources/images/banners/htmlcss-banner.jpg")
    p.blog-preview-title A Real Beginner's Guide to HTML and CSS
    p.blog-preview-text Skip the $17,000 bootcamp and join me on a self-taught journey :)
```
Notice how I'm using an un ordered list there are no divs. Also notice that I change it so only the image is wrapped with the `<a>` tag. This is more correct since the `<a>` is an inline tag, but a div and paragraph are block level elements. You should avoid wrapping block elements inside inline elements.